documentation/synthetic_tokens/security/security.md

```diff
@@ -0,0 +1,107 @@
+# Attack Vectors
```

Comment:
```suggestion
# Known Issues
```

Reply:
I think we should change the name of the file to `Known Issues.md`. 

Also, note the bug in the map.txt! You have this file in `synthetic_tokens/security/security.md` BUT in map.txt you have the file in `synthetic_tokens/security.md`. This will fail. (This is the same bug we had the other day).

I think the file should be `synthetic_tokens/known_issues.md`.

Reply:
Great, we can make those changes.